spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts Controller', type: :request do
+  describe '/index' do
+    before :each do
+      get '/users/:user_id/posts/'
+    end
+
+    it 'Check http success' do
+      get '/posts/index'
+      expect(response).to have_http_status(:ok)
+    end
+
+    it 'check render index' do
+      expect(response).to render_template(:index)
+    end
+
+    it 'Check the placeholder' do
+      expect(response.body).to include('Here is a list of all posts for given user')
+    end
+  end
+
+  describe '/show' do
+    before :each do
+      get '/users/:user_id/posts/:id'
+    end
+
+    it 'Check http success' do
+      get '/posts/show'
```

Comment:
- [ ] Please delete this line. It throws an error because this rou
![image](https://user-images.githubusercontent.com/45460192/206705736-d8699782-6465-483c-b105-bc97210226b7.png)
te does not exist 👍